IFC4x3/PropertyEnumerations/s/PEnum_SleeperArrangement.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_SleeperArrangement" Name="PEnum_SleeperArrangement" UniqueId="79fbe707-00b7-4903-b424-246fce743dfb">
```

Comment:
Does it require documentation?

Reply:
de4bcf9de7f5e2e1d9e24e66e3b338da3b5506eb